### PR TITLE
IN-D Invoice Data Capture

### PR DESCRIPTION
**Pull request by gayathri35**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by gayathri35** [comment]
Hello @sriyen-msft ,
Can you please review this connector and I need to pulish this connector ISV studio as well, So I am requesting to the approve my connector.

File: certified-connectors/Payables E2E/README.MD [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# IN-D Invoice Data Capture
```
Comment (v-EgorKozhadei) [inline certified-connectors/Payables E2E/README.MD:1]: @gayathri35 please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1). All of the required headings should be present.

File: certified-connectors/Payables E2E/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,132 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Payables E2E/apiDefinition.swagger.json:4]: @gayathri35 please change the version to `1.0`

File: certified-connectors/Payables E2E/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    "title": "IN-D Invoice Data Capture",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Payables E2E/apiDefinition.swagger.json:5]: @gayathri35 is this title correct? The PR is named `IN-D Payables E2E` and the folder's name is `Payables E2E`.

File: certified-connectors/Payables E2E/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    "description": "A smart invoice processing solution that converts static documents (scanned, PDFs, screenshots, etc.) into usable information to fuel your business processes.IN-D Payables is an AI led intelligent document classification and capture solution that automates processing of all kinds of commercial documents used in Shipping, Logistics and Manufacturing operations. Manual handling of documents slows the pace of business transactions."
```
Comment (v-EgorKozhadei) [inline certified-connectors/Payables E2E/apiDefinition.swagger.json:6]: @gayathri35 please make sure that everything is grammatically correct and is free of typographical errors. 
Connector description should describe concisely the main purpose and value offered by your connector.

File: certified-connectors/Payables E2E/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/api/users/token/": {},
```
Comment (v-EgorKozhadei) [inline certified-connectors/Payables E2E/apiDefinition.swagger.json:18]: @gayathri35 empty operations are [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).

File: certified-connectors/Payables E2E/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+    "/api/e2e": {
+      "post": {
+        "summary": "Payable E2E",
+        "description": "Process invoices to extract the key attributes like invoice numbers, PO number, vendor name, etc., and export the data",
+        "operationId": "Payable_E2E",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Payables E2E/apiDefinition.swagger.json:23]: @gayathri35 operationId properties [should be in pascal case](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers) and without hyphens or underscores.

File: certified-connectors/Payables E2E/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+                "files"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline certified-connectors/Payables E2E/apiDefinition.swagger.json:96]: @gayathri35 default responses with an empty schema are [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses), as well as using only default schema for responses. Please define possible responses for your operations. You can refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @gayathri35!

Please check comments after the review.

Let us know if you have any questions.

**Comment by v-EgorKozhadei** [comment]
Hello @gayathri35, 

I hope you are doing well. 

I am following up with you to see how things are going with updating your connector.
I noticed that you changed only a few things from the comments, the rest hasn't been fixed yet.
After changing connector's and PR's titles, please change the name of connector folder so that it matches connector's title.

Please let us know if you have any questions or need help. 

Thank you.

**Comment by gayathri35** [comment]
Hello @v-EgorKozhadei,
Yes, I am following your comments and updating the connector. Thanks for your support.

**Pull request closed without merging**